src/compojure/api/api.clj

```diff
@@ -50,14 +52,19 @@
   [& body]
   (let [[options handlers] (common/extract-parameters body false)
         options (rsc/deep-merge api-defaults options)
-        handler (apply c/routes (concat [(swagger/swagger-routes (:swagger options))] handlers))
+        raw (:raw options)
+        swagger-ui (-> (swagger/swagger-routes (:swagger options))
+                       (cond-> raw ;; if the api won't be wrapped, swagger-ui still needs to be
```

Comment:
Yes, I mean now that you've told me there's a way to get just the swagger routes separately, without messing with the api middlewares, I think everything is just simpler. It was harder to add this feature when I thought I had to work it into `api`.